alacritty_terminal/src/term/cell.rs

```diff
@@ -57,6 +57,18 @@ impl Hyperlink {
     }
 }
 
+impl From<vte::ansi::Hyperlink> for Hyperlink {
```

Comment:
Should be imported as `VteHyperlink` at the top.

Reply:
Fixed in https://github.com/alacritty/alacritty/pull/6896/commits/d067434b45ddb26b5dd181a98d6fc022962a9a51.